include/SDL3/SDL_stdinc.h

```diff
@@ -1291,6 +1291,22 @@ extern SDL_DECLSPEC int SDLCALL SDL_strcasecmp(const char *str1, const char *str
  */
 extern SDL_DECLSPEC int SDLCALL SDL_strncasecmp(const char *str1, const char *str2, size_t maxlen);
 
+/**
+ * Scans a null-terminated string for any character from a null-terminated
+ * string, and returns a pointer to that character.
```

Comment:
This is again a matter of personal preference, but I find the description a bit ambiguous/hard to understand.

I've come up with this:

    Searches a string for the first occurence of any character contained in a breakset, and returns a pointer from the string to that character.
    \param str The null-terminated string to be searched.
    \param breakset A null-terminated string containing the list of characters to look for.
    \returns A pointer to the location, in str, of the first occurence of a character present in the breakset, or NULL if none is found.

I called the second parameter "breakset" because it helped with my wording.

I don't think my version is perfect, but I wanted to start a discussion about how it should be written.

Reply:
Your text is less ambiguous then mine, so I changed it to your suggestion. Thanks :)